src/main/java/com/ui/DashboardPage.java

```diff
@@ -77,5 +82,27 @@ public void selectProject(String projectName) {
         }
 
     }
+    public boolean dashboardDeleted(String dashName)
+    {
+        String projectName="raju1284_personal";
```

Comment:
The project name is hard-coded, could you please move this as a method argument/param for more flexibility.